index.js

```diff
@@ -303,27 +307,26 @@ module.exports = (api, options) => {
       let queuedBuilds = 0
       // Electron process
       let child
-      // Auto restart flag
-      let childRestartOnExit = 0
-      // Graceful exit timeout
-      let childExitTimeout
       // Function to kill Electron process
       const killElectron = () => {
         if (!child) {
           return
         }
 
+        const currentChild = child
```

Comment:
This is mostly what is fixing the issue. I also simplified the logic around the child process exit handling. 😸️